.gitignore

```
@@ -0,0 +1,3 @@
+/locales/*
+/node_modules
+.vscode/
```

Please add traling new lines to avoid diff noise. You can configure your editor to do this automatically.
Please see `insert_final_newline` https://editorconfig.org/ 

```suggestion
.vscode/

```

---

Bump.